src/dnddiceroller/DiceTower.java

```diff
@@ -15,14 +15,37 @@
  * @author Paul Scarrone
  */
 public class DiceTower {
-  final int PANEL_COUNT = 3;
-  List<Die> dice;
-
-  public DiceTower() {
-	this.dice = new ArrayList();
-  }
+  final int PANEL_COUNT = 3;    // holds the number of panels in the dice tower
+  List<Die> dice;               //
+  int trayValue;
+    
+    public DiceTower() {
+	this.dice = new ArrayList(); // 
+        this.trayValue = 0;
+    }
   
   public DiceTower(List dice) {
-	this.dice = dice;
+	this.dice = dice; // 
+        this.trayValue = 0;
   }
-}
+
+    public int getTrayValue() {
+        return trayValue;
+    }
+
+    public void dropDice() {
+        for(int panels = 0; panels < PANEL_COUNT; panels++){
+        int i = 0;
+        dice.stream().forEach((die) -> {
+            die.roll();    
+        });
+        trayValue = dice.get(i).getValue();
```

Comment:
Perhaps I misunderstood the concept of a dice tower. I thought the objective was to have a new roll for each panel.